anta/tests/connectivity.py

```diff
@@ -171,6 +181,8 @@ class Input(AntaTest.Input):
 
         neighbors: list[LLDPNeighbor]
         """List of LLDP neighbors."""
+        strict_mode: bool = False
```

Comment:
This is the default behavior today.
```suggestion
        strict_mode: bool = True
```